deployment/ansible/templates/epoch.yaml.j2

```diff
@@ -8,7 +8,7 @@ peers:
 
 http:
     external:
-        peer_address: {{ ansible_default_ipv4.address }}
+        peer_address: http://{{ ansible_default_ipv4.address }}:3013/
```

Comment:
I see this PR does not include the merged genesis PR though I do not expect conflict https://github.com/aeternity/epoch/pull/429/files#diff-d7503bfba48dd964db179e7dc45da770R15